code/datums/mutations/touch.dm

```diff
@@ -28,24 +28,16 @@
 	inhand_icon_state = "zapper"
 
 /obj/item/melee/touch_attack/shock/afterattack(atom/target, mob/living/carbon/user, proximity)
-	if(!proximity)
+	if(!proximity || !isliving(target))
 		return
-	if(iscarbon(target))
-		var/mob/living/carbon/C = target
-		if(C.electrocute_act(15, user, 1, SHOCK_NOGLOVES | SHOCK_NOSTUN))//doesnt stun. never let this stun
+	var/mob/living/L = target
```

Comment:
give em proper and descriptive names please

Reply:
ok